### PR TITLE
Updated to UniversalIO

### PR DESCRIPTION
**Pull request by NextdoorPsycho**
This adds the Universal IO package, then updates your imports.
its the same thing, but it works on all platforms.
Cheers!

### CODE REVIEW

File: lib/src/widgets/adw/header_bar.dart [hunk 1]
```diff
@@ -215,15 +202,11 @@ class _AdwHeaderBarState extends State<AdwHeaderBar> {
...
-                              sep != null &&
-                              sep[0].split(',').isNotEmpty) ...[
+                          if (hasWindowControls && sep != null && sep[0].split(',').isNotEmpty) ...[
                             SizedBox(width: widget.style.titlebarSpace),
                             for (var i in sep[0].split(','))
                               if (windowButtons[i] != null) windowButtons[i]!,
-                            if (!widget.style.nativeControls ||
-                                !kIsWeb && Platform.isLinux)
-                              SizedBox(width: widget.style.titlebarSpace),
+                            if (!widget.style.nativeControls || !kIsWeb && Platform.isLinux) SizedBox(width: widget.style.titlebarSpace),
```
Comment (prateekmedia) [inline lib/src/widgets/adw/header_bar.dart:209]: Can you please format this file with line characters as 80, see this for more:
https://dart.dev/tools/linter-rules/lines_longer_than_80_chars

File: lib/src/widgets/adw/new/scaffold.dart [hunk 2]
```diff
@@ -13,15 +13,14 @@ class AdwScaffold extends StatefulWidget {
     this.flapStyle,
     this.flapOptions,
     this.flapController,
-    @Deprecated('headerbar is deprecated, use the properties separately')
-        AdwHeaderBar? Function(Widget?)? headerbar,
+    @Deprecated('headerbar is deprecated, use the properties separately') AdwHeaderBar? Function(Widget?)? headerbar,
```
Comment (prateekmedia) [inline lib/src/widgets/adw/new/scaffold.dart:16]: format this too again.

**Review by prateekmedia — CHANGES_REQUESTED** [review]

**Comment by prateekmedia** [comment]
@NextdoorPsycho The PR looks good but the formatting is wrong, please format with respect to line limit 80 characters.

**Comment by NextdoorPsycho** [comment]
You could literally just merge it and make a version bump on your IDE and
just visiting the page will make it auto format to 80.

**Review by prateekmedia — APPROVED** [review]

**Pull request merged**